packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -119,8 +130,22 @@ const hasValidParticipant = (chatMessage: ChatMessageWithStatus): boolean =>
  * @public
  */
 export const messageThreadSelector: MessageThreadSelector = createSelector(
-  [getUserId, getChatMessages, getLatestReadTime, getIsLargeGroup],
-  (userId, chatMessages, latestReadTime, isLargeGroup) => {
+  [getUserId, getChatMessages, getLatestReadTime, getIsLargeGroup, getReadReceipts, getParticipants],
+  (userId, chatMessages, latestReadTime, isLargeGroup, readReceipts, participants) => {
+    // get number of participants
+    // -1 because the first object in participants is not a valid participant (no display name)
+    const messageThreadParticipantCount = Object.values(participants).length - 1;
```

Comment:
The no displayname user is the moderator from our sample app server - maybe -1 is not generic enough, let's filter all empty displayName participant either here or in component

Reply:
Added filtering instead of -1